### DISCUSSION TITLE
Batch case configuration

### DISCUSSION THREAD

**Discussion by gocfd** [nextfoam/baram#111] [Q&A] [answered]
Hello,

can you please help me to correctly configure batch case? I would like to change mass flow as an input parameter and monitor static pressure as an output parameter.

Thank you.

**Comment by kdg5424** [comment]
Hello @gocfd. This is Donggyu from NEXTFOAM.

Could you check the parameter is written in **Capital language**?
It should be written in Captial langauge.
And you should make a .csv file for the parameter. It should be like the below picture.

![image](https://github.com/user-attachments/assets/1d49eb6b-990c-4daa-9c85-17cb3a27b1ed)


Best regards

Donggyu kim

**Comment by gocfd** [comment]


**Reply by kdg5424** [reply]
Hello Gasper. I was mistaken. I recommend you to use **Capital Language** for the parameter.

Thanks

**Comment by gocfd** [comment]
Hello,

I did exactly as you described on Github. However, when I try to import
.csv file to Batch case nothing happens. The file is not shown in the table.
[image: image.png]

Best regards,

V V pet., 16. avg. 2024 ob 09:03 je oseba kdg5424 ***@***.***>
napisala:

**Comment by kdg5424** [comment]
Hello.
I can't see the picture what you uploaded.
Could you upload again?

Best regards

**Comment by kdg5424** [comment]
Hello.

I still can't see the picture.
Anyway could you try to run BARAM again with adminstrator mode?

Regards

**Comment by gocfd** [comment]
Hello,

I did run Baram as administrator.
![Screenshot 2024-08-16 091713](https://github.com/user-attachments/assets/f99dbe1c-4d38-4981-a2d0-54f887116826)

Best regards

**Comment by kdg5424** [comment]
Hello. Then, is everything fine with your case?

Regards.

**Comment by jiban** [comment]
Hi @gocfd ,
Could you share the `CSV` file you used? (i.e. `batch_case_test.csv` file )

**Comment by gocfd** [comment]
Hello,

[batch_case_test.csv](https://github.com/user-attachments/files/16634700/batch_case_test.csv)

**Comment by jiban** [comment]
Dear @gocfd,
The *separator* should be `,`(comma) rather than `;`(semicolon) as *CSV* stands for  *Comma Separated Values*.
Please refer to following files. (Right one is correct one)

![image](https://github.com/user-attachments/assets/d0a17730-b177-47c6-bb64-efd16d0f2162)

**Comment by gocfd** [comment]
Hello,

great, now it works! However, when I want to click Start Calculation pop up window appears: 
![image](https://github.com/user-attachments/assets/922bbcf0-cce2-44cb-ac10-e176cde89c26)

Best regards

**Comment by jiban** [comment, marked as answer]
Only scheduled cases will join the calculation.
Select case(s) in the list by mouse click (and optionally CTRL or SHIFT key) as you usually do in Windows File Explorer, and click-click mouse button.
This will show context menu like following.

![image](https://github.com/user-attachments/assets/898bc856-6088-4d4e-9007-d3a0d7300978)

Once scheduled, you can see the check mark in *Calc.* column.

![image](https://github.com/user-attachments/assets/fd52cadb-a82a-4eaf-b911-c5c6c42e6107)

Only those checked cases will be calculated in order if you click *Start Calculation* button.

**Reply by gocfd** [reply]
Great, now it works. Many thanks! 

Best regards,
